webapp/template_utils.py

```diff
@@ -79,6 +79,30 @@ def static_url(filename):
     return url + "?v=" + file_hash.hexdigest()[:7]
 
 
+def install_snippet(
+    package_name, lowest_risk_available, default_track, confinement
+):
+    """
+    Template function that returns the snippet value to
+    install a snap to be used in distro pages and/or snap
+    detail pages
+    """
+
+    snippet_value = "sudo snap install " + package_name
+
+    if default_track != "latest":
+        snippet_value += (
+            " --channel=" + default_track + "/" + lowest_risk_available
```

Comment:
This could be snippet_value = f" --channel={default_track}/{lowest_risk_available}"

Reply:
👍